linux_os/guide/system/software/integrity/fips/enable_fips_mode/rule.yml

```diff
@@ -59,4 +59,4 @@ warnings:
         See <b>{{{ weblink(link="http://csrc.nist.gov/groups/STM/cmvp/documents/140-1/1401vend.htm") }}}</b>
         for a list of FIPS certified vendors.
 
-platform: machine
+platform: kernel
```

Comment:
You can install the kernel package in a container. This rule probably still shouldn't be applicable in that case, because the kernel on the host should handle FIPS and the kernel in the container won't run.